docs/ova/README.md

```diff
@@ -0,0 +1,84 @@
+# Cluster API Provider vSphere, OVA Installer Build process
+
+The build process uses a collection of bash scripts to launch a docker container on your local machine
+where we provision a linux OS, install dependencies, and extract the filesystem to make the OVA.
+
+- [Cluster API Provider vSphere, OVA Installer Build process](#cluster-api-provider-vsphere-ova-installer-build-process)
+  - [Usage](#usage)
+    - [Prerequisites](#prerequisites)
+    - [Build bundle and OVA](#build-bundle-and-ova)
+      - [Build script](#build-script)
+  - [Deploy](#deploy)
+    - [Using the bootstrap cluster](#using-the-bootstrap-cluster)
+
+## Usage
+
+The build process is controlled from a central script, `build.sh`. This script
+launches the build docker container and controls our provisioning and ova
+extraction through `make`.
+
+### Prerequisites
+
+The build machine must have `docker`.
+
+- `docker for Mac`: https://www.docker.com/docker-mac
+- `docker for Windows`: https://www.docker.com/docker-windows
+
+### Build bundle and OVA
+
+#### Build script
+
+The build script pulls the desired versions of each included component into the build container.
+
+> You must specify build step `ova-dev` when calling build.sh
+
+If called without any values, `build.sh` will get Kubernetes version: `stable-1`
+```
+./build/build.sh ova-dev
+```
+
+If called with the values below, `build.sh` will include Kubernetes version v1.12.1
+```
+./build/build.sh ova-dev --kubernetes-version v1.12.1
+```
+
+## Deploy
+
```

Comment:
I think we should mention something about generating the key pairs here before they start.  The top level make file will generate a set of key files for them.  When they run `make dev-yaml`, it will generate vsphere_tmp and vsphere_tmp.pub in their .ssh folder.

Reply:
I thought this would be more appropriate in the general docs for the vSphere provider, this is strictly related to the OVA / Management cluster